vm_manager/helpers/pacemaker.py

```diff
@@ -345,15 +354,7 @@ def add_meta(self, key, value):
         Add a meta to the resource
         :param meta: the meta to add
         """
-        args = [
-            "crm",
-            "resource",
-            "meta",
-            self._resource,
-            "set",
-            key,
-            value
-        ]
+        args = ["crm", "resource", "meta", self._resource, "set", key, value]
```

Comment:
Don't change the way you construct the command just here, or change it everywhere

Reply:
This line was over 79 caracters.